app/views/pages/conditions/new.html.erb

```diff
@@ -25,7 +25,23 @@
       end %>
 
       <%= f.govuk_collection_select :answer_value, condition_input.routing_answer_options, :value, :label, label: { class: "govuk-!-font-weight-bold" }, options: { include_blank: t("helpers.label.pages_conditions_input.default_answer_value") } %>
-      <%= f.govuk_collection_select :goto_page_id, condition_input.goto_page_options, :id, :question_text, label: { class: "govuk-!-font-weight-bold" }, options: { include_blank: t("helpers.label.pages_conditions_input.default_goto_page_id") } %>
+      <% unless FeatureService.new(group: condition_input.form.group).enabled?(:exit_pages) %>
+        <%= f.govuk_collection_select :goto_page_id, condition_input.goto_page_options, :id, :question_text, label: { class: "govuk-!-font-weight-bold" }, options: { include_blank: t("helpers.label.pages_conditions_input.default_goto_page_id") } %> 
+      <% else %>
+        <%= f.govuk_select(:goto_page_id, label: { class: "govuk-!-font-weight-bold" }, options: { include_blank: t("helpers.label.pages_conditions_input.default_goto_page_id") }) do %>
+          <%= condition_input.goto_page_options.each do |option| %>
```

Comment:
`<%= condition_input.goto_page_options.each do |option| %>`

I spotted this by chance, when the equals is included the Openstruct with the options is turned into a string and included in the html output!

```html
[#&lt;OpenStruct id=43, question_text=&quot;3. n2&quot;&gt;, #&lt;OpenStruct id=44, question_text=&quot;4. s2&quot;&gt;, #&lt;OpenStruct id=45, question_text=&quot;5. n5&quot;&gt;, #&lt;OpenStruct id=46, question_text=&quot;6. n7&quot;&gt;, #&lt;OpenStruct id=&quot;check_your_answers&quot;, question_text=&quot;Check your answers before submitting&quot;&gt;]
```

Changing to `<% condition_input.goto_page_options.each do |option| %>` fixes it.